package.json

```diff
@@ -1,20 +1,20 @@
 {
   "name": "screeps-jest",
-  "version": "1.3.0",
+  "version": "2.0.0",
```

Comment:
Most Jest "plugins" (for the lack of a better word) follow the Jest versioning. Might be a good idea to do the same here and bump this to 27.0.0

Reply:
Since it probably ony works with that specific major version, I think that makes sense.